tasks/section1.yml

```diff
@@ -661,7 +661,7 @@
   notify: generate new grub config
   when:
       - rhel7cis_set_boot_pass|bool
-      - grub_pass is defined and grub_pass.passhash is defined and grub_pass.passhash != ''
+      - grub_pass is defined and grub_pass.passhash is defined and grub_pass.passhash
```

Comment:
Why? The previous was clearer. Was it incorrect?

Reply:
Not incorrect, but this is flagged by ansible-lint [rule 602](https://github.com/ansible/ansible-lint/blob/master/lib/ansiblelint/rules/ComparisonToEmptyStringRule.py)